### ISSUE TITLE
Missing files

### ISSUE THREAD

**Issue by robotechnics** [zhuyifei1999/guppy3#15]
sets directory is empty
Unable to import
from guppy.sets.setsc import BitSet        # base bitset type
from guppy.sets.setsc import ImmBitSet     # immutable bitset type
from guppy.sets.setsc import immbit        # immutable bitset singleton constructor
from guppy.sets.setsc import immbitrange   # immutable bitset range constructor
from guppy.sets.setsc import immbitset     # immutable bitset constructor
from guppy.sets.setsc import MutBitSet     # mutable bitset
from guppy.sets.setsc import NodeSet       # base nodeset type
from guppy.sets.setsc import ImmNodeSet    # immmutable nodeset type
from guppy.sets.setsc import MutNodeSet    # mutable nodeset type

**Comment by zhuyifei1999** [comment]
Did you install the package with pip?

If installed properly, either with pip or setup.py, sets should contain
setsc, a compiled extension module.

**Comment by robotechnics** [comment]
Thanks for your quick reply.
   
  
  
   I cannot use pip install as I am working on a restrictive environment and cannot add additional packages to my installation.
   
  
  
   I downloaded the files from github and tried to link it to my program. That's where I noticed the missing files in github. 
   
  
  
   
  
  
   Is there anyway to uploaded them into github?
   
  
  
   
  
  
   Thank you very much for your excellent project
   
  
  
   
  
  
   Cheers
   
  
  
   Alberto
   
  
  
    
  
  
   El 15 de enero de 2020 a las 8:01 zhuyifei1999 <notifications@github.com> escribió: 
   
   Did you install the package with pip? 
   
   If installed properly, either with pip or setup.py, sets should contain 
   setsc, a compiled extension module.
